Dart/src/com/jetbrains/lang/dart/ide/runner/server/vmService/DartVmServiceListener.java

```diff
@@ -80,7 +80,8 @@ public void received(@NotNull final String streamId, @NotNull final Event event)
         break;
       case PausePostRequest:
         // We get this event after an isolate reload call, when pause after reload has been requested.
-        // TODO: Set current breakpoints; resume the isolate.
+        // This adds the "supports.pausePostRequest" capability.
+        myDebugProcess.getVmServiceWrapper().restoreBreakpointsForIsolate(event.getIsolate().getId());
```

Comment:
I can add a `CountDownLatch` into restoreBreakpointsForIsolate. I don't think they'll be an issue however, as the requests into the isolate will be received in order (set bp 1, set bp 2, resume). Happy to do either -